LICENSE.note

```diff
@@ -0,0 +1,25 @@
+The eventstudyr package as a whole is distributed under MIT. The eventstudyr package includes one other open source software component that is distributed under MIT. The component is a GitHub repository called suptCriticalValue
```

Comment:
Is it necessary for us to ship `suptCriticalValue` with our code? Is this because `suptCriticalValue` is not itself on the CRAN, and so can't be automatically declared to CRAN as a dependency?

Reply:
> Is it necessary for us to ship suptCriticalValue with our code?

@jmshapir based on the last paragraph of this [section](https://r-pkgs.org/license.html#sec-how-to-include) (quoted below):

> If you’re submitting to CRAN and the bundled code has a different (but compatible) license, you also need to include a LICENSE.note file that describes the overall license of the package, and the specific licenses of each individual component. For example, the diffviewer package bundles six javascript libraries all of which use a permissive license. The [DESCRIPTION](https://github.com/r-lib/diffviewer/blob/main/DESCRIPTION) lists all copyright holders, and the [LICENSE.note](https://github.com/r-lib/diffviewer/blob/main/LICENSE.note) describes their licenses. (Other packages use other techniques, but I think this is the simplest approach that will fly with CRAN.)

I thought that we needed to include the `suptCriticalValue` license. It also uses the [MIT License](https://github.com/ryanedmundkessler/suptCriticalValue/blob/c1dd6b2e9aaeaa7990efa2c857e76806d23bea9e/LICENSE), but the copyright lines are obviously [different](https://github.com/ryanedmundkessler/suptCriticalValue/blob/c1dd6b2e9aaeaa7990efa2c857e76806d23bea9e/LICENSE#L3) so the 2 licenses aren't _exactly_ the same. 

> Is this because suptCriticalValue is not itself on the CRAN, and so can't be automatically declared to CRAN as a dependency?

Yes! We "cite" the other functions we use [here](https://github.com/JMSLab/eventstudyr/blob/6508d84e95b81100d71044f21132bcc086f3f4d3/NAMESPACE#L8-L22), but because it's not on CRAN it's more complicated to include the package (see the thread [here](https://stackoverflow.com/questions/33335321/include-non-cran-package-in-cran-package)).

I still like our approach of copying the `suptCriticalValue` source code just in case something happens to the repository, but if you'd like to take a different approach or think our current method overcomplicates the licensing, please let me know!

Reply:
@nateschor thanks. Given the complexities of declaring a dependency that points to a non-CRAN package I agree this seems like the best course of action.

It also seems like the first best would be if the author of the package would include their package in the CRAN. So, in https://github.com/ryanedmundkessler/suptCriticalValue/issues/2 I asked them if they'd like to do that.

I suggest we give them about a day to respond and if we don't hear anything we can proceed with the current approach.

Reply:
Per https://github.com/ryanedmundkessler/suptCriticalValue/issues/2#issuecomment-1501161684 I think we should go ahead and ship to CRAN. We can update later if `suptCriticalValue` ships there too.